test/unit/testDeceasedDod.js

```diff
@@ -14,27 +14,49 @@ describe('DeceasedDod', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
```

Comment:
Please update the description for this test as it is for deceased dod but it is referring to deceased address.